style/style.css

```diff
@@ -0,0 +1,1065 @@
+    padding-left:8%;
+    justify-content:space-between;
+}
+
+.workerCard{
+    height:100%;
+    width:140px;
+    padding-left: 10px;
+    padding-right: 10px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.workerPhoto{
+    height:50%;
+    width:100%;
+}
+
+.w1 .workerPhoto{
+    background: url("../img/worker1.png") no-repeat center top ;
+}
+
+.w2 .workerPhoto{
+    background: url("../img/worker2.png") no-repeat center top ;
+}
+
+.w3 .workerPhoto{
+    background: url("../img/worker3.png") no-repeat center top ;
+}
+
+.workerInfo{
+    height: 40%;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.workerInfo .name{
+    font-family: LatoS;
+    font-size: 17px;
+    color : #303030;
+}
+
+.workerInfo .position{
+    font-family: LatoR;
+    font-size: 15px;
+    color : #8e8e8e;
+}
+
+.workerInfo a
+{
+    color: #8e8e8e;
+    text-decoration: none;
+}
+
+.workerInfo .links{
+    width: 100%;
+    justify-content: space-around;
+    align-items: center;
+}
+.workerInfo a:hover{
+    color: #6a98d1;
+}
+
+/* comments */
+
+#slides {
+    position: relative;
+    height: 190px;
+    padding: 0px;
+    margin: 0px;
+    list-style-type: none;
+}
+
+.slide {
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    width: 100%;
+    height: inherit;
+    opacity: 0;
+    z-index: 1;
+
+    -webkit-transition: opacity 1s;
+    -moz-transition: opacity 1s;
+    -o-transition: opacity 1s;
+    transition: opacity 1s;
+}
+
+.showing {
+    opacity: 1;
+    z-index: 2;
+}
+
+.slide{
+    height: 100%;
+}
+
+.slide>.flex-column{
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    width: 70%;
+}
+
+.quote{
+    font-family: LatoL;
+    font-size: 20px;
+    color: #fff;
+    text-align:center;
+}
+
+.slidercontrol{
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+}
+
+.slidercontrol .status{
+    margin-top: 2%;
+    height: 100%;
+    width: 6%;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.slidercontrol>.name{
+    margin-top:2%;
+    font-family: LatoM;
+    font-weight: bold;
+    font-size: 20px;
+    color: #fff;
+    border-right: 1px solid #fff;
+    width:15%;
+}
+
+.slidercontrol .status .point{
+    width:8px;
+    height:8px;
+    border: 1px solid #fff;
+    border-radius: 50%;
+}
+
+.slidercontrol .status .point.set{
+    background-color: #fff;
+}
+
+.slide:nth-of-type(1) {
+    background: url("../img/Comments_BG.png") no-repeat center top / cover;
+}
+.slide:nth-of-type(2) {
+    background: url("../img/Comments_BG_2.png") no-repeat center top / cover;
+}
+.slide:nth-of-type(3) {
+    background: url("../img/Comments_BG_3.png") no-repeat center top / cover;
+}
+
+/* function */
+
+.functions>.flex-row{
+    justify-content: space-between;
+    align-items: center;
+}
+
+.funct{
+    width: 20%;
+    padding:3% 0%;
+    justify-content: center;
+    align-items: center;
+    margin: 5% 0px;
```

Comment:
You can make `margin: 8% 0` instead of adding 3% padding (padding doesn't have any difference from margin here)